README.md

```diff
@@ -20,17 +20,48 @@ pip install fx-crash-sig
 [Example script](/fx_crash_sig/example.py):
 
 ```py
-from fx_crash_sig import sample_traces
+import json
+
 from fx_crash_sig.crash_processor import CrashProcessor
 
+with open("crashping.json") as fp:
+    crash_ping = json.load(fp)
+
 crash_processor = CrashProcessor()
 
-signature = crash_processor.get_signature(sample_traces.trace1)
-    
+signature_result = crash_processor.get_signature(crash_ping)
+print(signature_result.signature)
```

Comment:
I want to get rid of `sample_traces.py` because it's a huge file that's pretty unwieldy and I'm pretty sure it's got things that are the wrong shape in it. In the meantime, I rewrote this example without it.